<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWid">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuFileNew"/>
    <addaction name="menuFileOpen"/>
    <addaction name="menuFileSave"/>
    <addaction name="menuFileSaveAs"/>
    <addaction name="menuFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="menuEditUndo"/>
    <addaction name="menuEditRedo"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuFileNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="menuFileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="menuFileSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="menuFileSaveAs">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="menuFileQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuEditRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="menuEditUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="menuHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
